Description: disable check new version
Bug: 602535
--- a/src/config.cpp
+++ b/src/config.cpp
@@ -53,7 +53,7 @@ Config::Config()
 	m_advLayoutDirectionRL = settings.value( "advanced/layoutltr", false ).toBool();
 	m_advAutodetectEncoding = settings.value( "advanced/autodetectenc", false ).toBool();
 	m_advExternalEditorPath = settings.value( "advanced/editorpath", "/usr/bin/kate" ).toString();
-	m_advCheckNewVersion = settings.value( "advanced/checknewver", true ).toBool();
+	m_advCheckNewVersion = false;
 	m_toolbarMode = (Config::ToolbarMode) settings.value( "advanced/toolbarmode", TOOLBAR_LARGEICONSTEXT ).toInt();
 	m_lastOpenedDir = settings.value( "advanced/lastopendir", "." ).toString();
 
--- a/src/dialog_setup.cpp
+++ b/src/dialog_setup.cpp
@@ -177,7 +177,7 @@ void DialogSetup::accept()
 	
 	// Autodetect encoding
 	Check_Need_Restart( boxAutodetectEncoding, &pConfig->m_advAutodetectEncoding, &need_restart );
-	pConfig->m_advCheckNewVersion = cbCheckForUpdates->isChecked();
+	pConfig->m_advCheckNewVersion = false;
 
 	// Layout direction management
 	bool layout_rl = boxLayoutDirectionRL->isChecked();
